### PR TITLE
Updated README for the new concept

### PR DESCRIPTION
**Pull request by VladVin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,109 +30,70 @@ To install TorchOk locally, run:
...
-- Slow downloaded S3 bucket: `s3://<bucket-name>/<dirpath>`. Volume size is needed to be set when you use S3 bucket. 
-  For other cases it can be omitted.
-- Fast FSx access: `fsx://<file-system-id>/<mount-name>/<directory>`. To create FSx filesystem follow 
-  [this instructions](https://aws.amazon.com/blogs/machine-learning/speed-up-training-on-amazon-sagemaker-using-amazon-efs-or-amazon-fsx-for-lustre-file-systems/)
 
-Example with S3:
+## Getting started
+The folder `examples/configs` contains YAML config files with some predefined training and inference configurations.
+### Train
+Let's train a basic CIFAR-10 classification model. You can check the default configuration `examples/configs/classification_cifar10.yml`, where the CIFAR-10 dataset and the classification task are used. The CIFAR-10 dataset will be automatically downloaded into your `~/.cache/torchok/data/cifar10` folder (341 MB).
```
Comment (PososikTeam) [inline README.md:43]: May be use word `For train` not `Let's train`?

File: README.md [hunk 2]
```diff
@@ -33,109 +30,70 @@ To install TorchOk locally, run:
...
-  save_dir: "s3://sagemaker-mlflow-main/mlruns"
-  secrets_manager:
-    region: "eu-west-1"
-    mlflow_secret: "acme/mlflow"
-```
+Losses
+- Pytorch Metric Learning losses
+- NT-ext (for unsupervised training)
+
+Metrics
```
Comment (PososikTeam) [inline README.md:96]: Torchmetrics already have it, if we need to say that we rewrite some metrics, i think we will use Torchmetrics metric.
